sr_hand_config/Template_EL/general_info.yaml

```diff
@@ -17,7 +17,7 @@ type: 'hand_e'
 version: 'E3M5'
 fingers: [th, ff, mf, rf, lf]
 sensors:
-  tip: {th: pst, ff: pst, mf: pst, rf: pst, lf: pst}
+  tip: {th: mst, ff: mst, mf: mst, rf: mst, lf: mst}
```

Comment:
Do we want to change the tactile type in the templates?
Does this mean that from now on we're selling MST hands by default?

Reply:
Reverting these changes for now, until we speak with production